src/box/lua/tuple.c

```diff
@@ -757,6 +757,30 @@ lbox_tuple_info(lua_State *L)
 	return 1;
 }
 
+/**
+ * Push to Lua stack a table with the information about a format of a tuple.
+ * Elements of a table are structs {'name', 'type'}.
+ */
```

Comment:
So is the implementation should be similar to `lbox_tuple_to_map`:

https://github.com/tarantool/tarantool/blob/c5b3e5949b43131276b492e59dec5a41fa49573a/src/box/lua/tuple.c#L499-L555

But create a table-array with {name = 'field_name', type = 'field_type'} instead of table-map?

Reply:
> So is the implementation should be similar to `lbox_tuple_to_map`:
> 
> https://github.com/tarantool/tarantool/blob/c5b3e5949b43131276b492e59dec5a41fa49573a/src/box/lua/tuple.c#L499-L555
> 
> But create a table-array with items `{name = 'field_name', type = 'field_type'}` instead of the table-map `{name/idx = value}`?

I think you can reuse the implementation of `tuple_format_serialize` here:
https://github.com/tarantool/tarantool/blob/c5b3e5949b43131276b492e59dec5a41fa49573a/src/box/lua/tuple_format.c#L112-L128

Reply:
Or can I return the `format:totable()` for the format I got? Will it be ok?

Reply:
Fixed, this function now pushes a table for real. I kept the current comment, is it ok?

Reply:
It still confuses me. Something like this will be more clear:
```
* Returns an array with the information about a format of a tuple.
* Elements of the array are structs {'name' = field_name, 'type' = field_type}.
```

Reply:
It still confuses me. Something like this will be more clear:
```
* Returns an array with the information about a format of a tuple.
* Elements of the array are structs {'name' = field_name, 'type' = field_type}.
```

Reply:
Changed this comment.